src/transformers/AFETransformer.ts

```diff
@@ -0,0 +1,35 @@
+import { FormSchemaTransformer, OHRIFormField, OHRIFormSchema } from '../api/types';
+
+export const AngularFormTransformer: FormSchemaTransformer = {
```

Comment:
I would rename to something like: `AngularFormEngineSchemaTransformer` and also rename the file to: `angular-fe-schema-transformer.ts`